PyDSS/pyPostprocessor/PostprocessScripts/EdLiFoControl.py

```diff
@@ -0,0 +1,579 @@
+#**Authors: ksedzro@ieee.org**
+
+
+# Additional packages
+import os
+import matplotlib.pyplot as plt
+import pandas as pd
+import math
+import opendssdirect as dss
+import networkx as nx
+import time
+import json
+import numbers
+# from place_new_regs import place_new_regs
+import numpy as np
+import seaborn as sns
+import scipy.spatial.distance as ssd
+from sklearn.cluster import AgglomerativeClustering
+import matplotlib.image as mpimg
+from PyDSS.pyPostprocessor.PostprocessScripts.postprocess_thermal_upgrades import postprocess_thermal_upgrades
+from PyDSS.utils.utils import iter_elements, check_redirect
+from PyDSS.pyPostprocessor.pyPostprocessAbstract import AbstractPostprocess
+from PyDSS.exceptions import InvalidParameter, OpenDssConvergenceError
+from PyDSS.utils.timing_utils import timed_info
+
+
+plt.rcParams.update({'font.size': 14})

```

Comment:
Would it be better to do this in a way that it gets reset once you're done using it? Seems like this would change the behavior for anyone else using it inside this application.

Reply:
Let's talk when you can.

Reply:
Per discussion, delete